README.md

```diff
@@ -73,6 +73,59 @@ pip install -r requirements/gui.txt
 After this, the command `python joinmarket-qt.py` from within the `scripts` subdirectory should work.
 There is a [walkthrough](docs/JOINMARKET-QT-GUIDE.md) for what to do next.
 
+#### Joinmarket-Qt desktop icon for Linux
+
+If you are using Ubuntu desktop and wish to use JoinMarket's graphical interface, the following instructions will create an icon in your Applications for easy access.
```

Comment:
Ubuntu isn't important, `/usr/share/pixmaps/` and `desktop-file-install` is a part of freedesktop.org standards, should work with almost any Linux distro and GUI.